src/main/java/robot/subsystems/drivetrain/commands/PurePursue.java

```diff
@@ -219,6 +220,20 @@ private double distanceLookahead() {
                 Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));
     }
 
+    /**
+     * Takes the current output, and gradually raises it to the target velocity, while making sure it doesn't change
+     * in a rate that is faster than the maximum acceleration rate.
+     *
+     * @param input the target velocity.
+     * @param lastOutput current output.
+     * @param limitRate maximum acceleration rate
+     * @return returns an updated output.
+     * @author Paulo
+     */
+    public double limitRate(double input, double lastOutput, double limitRate) {
+        double maxChange = 0.02 * limitRate;
```

Comment:
Consider putting `0.02` in a constant in case we want to change it. `CYCLE_TIME` could be a nice name for it.